code_schemes/icraf_s01e01.json

```diff
@@ -3,6 +3,188 @@
     "Name": "ICRAF s01e01",
     "Version": "0.0.0.1",
     "Codes": [
+      {
+        "CodeID": "code-94f7d4f7",
+        "DisplayText": "Tree planting",
```

Comment:
```suggestion
        "DisplayText": tree planting",
```

Reply:
Actually I see many of your other codes start with upper case. I think "burying waste" is the only that doesn't. Convention on other projects has to been to have all normal codes in lower case and control codes in upper case, but at least be consistent.